apps/authentication/const.py

```diff
@@ -31,6 +31,7 @@ def get_prop_backends(cls, confirm_type):
 class MFAType(TextChoices):
     OTP = 'otp', _('OTP')
     SMS = 'sms', _('SMS')
+    Email = 'email', _('Email')
     Face = 'face', _('Face Recognition')
     Radius = 'otp_radius', _('Radius')
     Custom = 'mfa_custom', _('Custom')
```

Comment:
There are no significant issues with this code, it looks very clean and well-written. No need to optimize.

For an improvement suggestion:
The `MFAType` class should be removed since it is not used anywhere in the function `get_prop_backends`. It can simply return all MFA types like so:

```python
def get_prop_backends(cls, confirmed_type):
    mfa_types = [(MOTA, _('MFA'))]
    
    if confirmed_type == 'otp':
        mfa_types.append((OTP, _('OTP')))
    elif confirmed_type == sms:
        mfa_types.append((SMS, _('SMS')))
    
    # Add more configurations as needed
    
    return mfa_types
```
This will make the code cleaner and reduce unnecessary complexity.